modules/tools/migration/src/main/java/org/wso2/mb/migration/DBConnector.java

```diff
@@ -292,17 +294,31 @@ void updateQueueNamesInQueuesAndBindings() throws SQLException {
                 String bindingData = bindingsResultSet.getString(BINDING_DETAILS);
                 if (queueNameHasCapitals(queueName)) {
                     String newQueueName = queueName.toLowerCase();
-                    bindingData = bindingData.replaceAll(queueName, newQueueName);
+                    bindingData = bindingData.replaceAll(String.format("queueName=%s", queueName), String.format
+                            ("queueName=%s", newQueueName));
```

Comment:
Maybe we can extract a method for code `String.format("queueName=%s", queueName)`. See it repeating in many places.